src/lib/JwtSessionManager.ts

```diff
@@ -202,6 +201,19 @@ export class JwtSessionManager<U extends ExpirableJwtValue> {
       return;
     }
 
+    const now: number = Date.now();
+    const lastUserRefreshTimestamp: string | null = localStorage.getItem(LOCAL_STORAGE_LAST_USER_REFRESH_TIMESTAMP);
+    // On ne refresh la session que si le dernier call n'a pas dépassé le temps du refreshToken, pour éviter que les
+    // deux appels rentrent en concurrence et qu'il y ai une incohérence de données
+    if (
+      lastUserRefreshTimestamp
+      && (now - parseInt(lastUserRefreshTimestamp, 10)) < (currentSession.refreshDurationInMillis / 2)
+    ) {
+      logger.debug('The user session is not refreshed as the last refresh call is done in less than 1 second');
```

Comment:
Je me rends compte que le message ici n'est pas bon, ce serait plutôt : `User session refresh aborted because it seems that it has already been refreshed recently in another tab`

Reply:
done